controllers/StudentController.js

```diff
@@ -0,0 +1,78 @@
+const StudentModel = require('../models/StudentModel')
+
+class StudentController {
+
+  static getStudents(req, res) {
+    StudentModel.findAll((err, data) => {
+      if (err) {
+        res.render('public/404', {errMsg: err})
+      } else {
+        res.render('student', {data})
+      }
+    })
+  }
+
+  static getStudentByEmail(req, res) {
+    StudentModel.findByEmail(req.url.split('/')[1], (err, data) => {
+      if (err) {
+        res.render('public/404', { errMsg: err })
+      } else if (typeof data === 'object') {
+        res.render('student/detail', { data })
+      } else {
+        res.render('public/404', { errMsg: data })
+      }
+    })
+  }
+
+  static getAdd(req, res) {
+    res.render('student/add')
+  }
+
+  static postAdd(req, res) {
+    StudentModel.createOne(req.body, (err, data) => {
+      if (err) {
+        res.render('public/404', { errMsg: err })
+      } else if (data === 'success') {
+        res.redirect('/students')
+      } else {
+        res.render('public/404', { errMsg: data })
```

Comment:
Kalau error validasi `'All fields should not be empty.'` diredirect page student/add aja ya, jadi tetep di form add student itu. nanti pesan error validasinya jg tampi disitu